scripts/oldump.sh

```diff
@@ -102,8 +102,9 @@ ls -lhR
 function archive_dumps() {
     # copy stuff to archive.org
     # TODO: Switch to ia client tool. This will only work in production 'til then
-    python /olsystem/bin/uploaditem.py $dump --nowait --uploader=openlibrary@archive.org
-    python /olsystem/bin/uploaditem.py $cdump --nowait --uploader=openlibrary@archive.org
+     ia $dump $dump.txt.gz --metadata "uploader:openlibrary@archive.org" --metadata "title:$dump" --metadata "collection:ol_exports"
+     ia $cdump $cdump.txt.gz  --metadata "uploader:openlibrary@archive.org" --metadata "title:$cdump" --metadata "collection:ol_exports"
+
```

Comment:
```suggestion
     log "ia version is v$(ia --version)"  # ia version is v1.9.0
     # Need to pre-generate `${dump}_files.xml` and `${dump}_meta.xml` via a subset of `uploaditeems.py`
     ia upload $dump $dump --metadata "uploader:openlibrary@archive.org" --metadata "title:$dump" --metadata "collection:ol_exports"
     ia upload $cdump $cdump --metadata "uploader:openlibrary@archive.org" --metadata "title:$cdump" --metadata "collection:ol_exports"
```

Reply:
I think the xml is generated though the metadata argument in the ia tool..will add the log call

Reply:
`$dump` and `$cdump` are identifiers and _directory_ names, not filenames.  We want to transfer all the files in those directories.